introduction/modules/modules.md

```diff
@@ -15,41 +18,51 @@ Connectors are Valtimo configurable components that can be used to connect to ex
 framework to create, configure, and use connectors. Certain Valtimo modules include their own connectors that become
 available automatically for configuration when starting a Valtimo instance.
 
+> **_NOTE:_** Connectors will be replaced by Plugins in the future
+
 ### Contract
 
-Contract contains certain interfaces and events that Valtimo uses in several other modules. Apart from this, contract
-sanitizes endpoint output when exceptions occur. A liquibase runner is available that allows other modules to define
-liquibase configuration that is run on startup. Finally, Contract includes several helper classes, like the
-`QueryDialectHelper` bean that can be used in criteria queries to add JSON column clauses to the query.
+Contract contains certain interfaces and events that Valtimo uses in several other modules. Consider it the 'glue'
+between a module and Core.
+
+Apart from this, contract sanitizes endpoint output when exceptions occur. A liquibase runner is available that allows
+other modules to define liquibase configuration that is run on startup. Finally, Contract includes several helper
+classes, like the `QueryDialectHelper` bean that can be used in criteria queries to add JSON column clauses to the
+query.
 
 ### Core
 
-Core contains and builds upon the Camunda process engine that enables Valtimo to run processes. Several endpoints and
-functions are available that provide task data and process data. It also includes choice fields. A generic way to create
-lists of values, that can be used by the application. Some security features used to control access to any Valtimo
-endpoints are also part of this module.
+Core is the main module of Valtimo. It contains and builds upon the Camunda process engine that enables Valtimo to run
+processes. Several endpoints and functions are available that provide task data and process data.
+
+It also includes choice fields. A generic way to create lists of values, that can be used by the application. Some
```

Comment:
Maybe provide an example as to what choice fields can be used for?

Reply:
Not sure if choice fields are so important that we need to mention it at all actually. However, it was already there and I applied your suggestion.